x-pack/solutions/observability/plugins/apm/public/components/shared/summary/transaction_summary.tsx

```diff
@@ -25,13 +26,14 @@ interface Props {
 
 function getTransactionResultSummaryItem(transaction: Transaction) {
   const result = transaction.transaction.result;
-  const url = transaction.url?.full || transaction.transaction?.page?.url;
+  const method = transaction.http?.request?.method;
+  const status = transaction.http?.response?.status_code;
+  const urlFull = transaction.url?.full || transaction.transaction?.page?.url;
 
-  if (url) {
-    const method = transaction.http?.request?.method;
-    const status = transaction.http?.response?.status_code;
+  const urlFromFields = buildUrl(transaction, urlFull);
 
-    return <HttpInfoSummaryItem method={method} status={status} url={url} />;
+  if (urlFromFields || method || status) {
+    return <HttpInfoSummaryItem method={method} status={status} url={urlFromFields} />;
```

Comment:
Should we really not render the `HttpInfoSummaryItem` component at all when any one of its properties are undefined? Maybe this component could accept its properties to be undefined and only render what's defined.